components/resources/library/src/iosMain/kotlin/org/jetbrains/compose/resources/ResourceReader.ios.kt

```diff
@@ -30,7 +41,11 @@ internal actual fun getPlatformResourceReader(): ResourceReader = object : Resou
 
     private fun readData(path: String, offset: Long, size: Long): NSData {
         val fileHandle = NSFileHandle.fileHandleForReadingAtPath(path) ?: throw MissingResourceException(path)
-        fileHandle.seekToOffset(offset.toULong(), null)
+        if (available(OS.Ios to OSVersion(major = 13))) {
+            fileHandle.seekToOffset(offset.toULong(), null)
```

Comment:
I do understand that the code will be tricky, but handling errors when obj-c methods provide them, is a good practice:
```
        memScoped {
            val errorWrapper = alloc<ObjCObjectVar<NSError?>>()
            fileHandle.seekToOffset(offset.toULong(), errorWrapper.ptr).also {
                errorWrapper.value?.let {
                    error(it.localizedDescription) // Throw specific exception here
                }
            }
        }
```